back-end/src/main/java/admin/exception/ControllerAdvice.java

```diff
@@ -21,12 +24,27 @@
         return ResponseEntity.badRequest().body(response);
     }
 
+    @ExceptionHandler(UnauthorizedException.class)
+    public ResponseEntity<ExceptionResponse> unauthorizedHandle(UnauthorizedException exception) {
+        ExceptionResponse response = ExceptionResponse.of(exception.getMessage());
+        return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(response);
+    }
+
     @ExceptionHandler(NotFoundException.class)
     public ResponseEntity<ExceptionResponse> notFoundHandle(NotFoundException exception) {
         ExceptionResponse response = ExceptionResponse.of(exception.getMessage());
         return ResponseEntity.status(HttpStatus.NOT_FOUND).body(response);
     }
 
```

Comment:
StatusCode 별로 Custom Exception을 사용할 거라면, CustomException에 status코드를 넣어주고 Handler를 묶어서 처리하는 건 어떨까요? 여러개의 예외를 잡을 필요 없이 묶어서 사용할 수 있을 것 같아 제안합니다! :)
```suggestion
    @ExceptionHandler(CustomException.class)
    public ResponseEntity<ExceptionResponse> handle(CustomException exception) {
        ExceptionResponse response = ExceptionResponse.of(exception.getMessage());
        return ResponseEntity.status(exception.getStatusCode()).body(response);
    }
```

Reply:
오호. 좋은 생각 같아요! 적용시켜 보겠습니다!

Reply:
확인했어요! 감사합니다ㅎㅎ